docs/content/installation/postgres-operator.md

```diff
@@ -108,8 +108,17 @@ for deploying the PostgreSQL Operator. The [example file](https://github.com/Cru
 contains defaults that should work in most Kubernetes environments, but it may
 require some customization.
 
-For the [full reference]("#") for how these configuration parameters work,
-please read the [PostgreSQL Operator Installer Configuration Reference]("#")
+For the [full reference](<{{< relref "/installation/configuration.md">}}>) for how these configuration parameters work,
+please read the [PostgreSQL Operator Installer Configuration Reference](<{{< relref "/installation/configuration.md">}}>)
```

Comment:
It doesn't seem like we need to link the same thing twice in one sentence. My impression is the first link can go away.

```suggestion
For a detailed description of each configuration parameter, please read the
[PostgreSQL Operator Installer Configuration Reference](<{{< relref "/installation/configuration.md">}}>)
```